querybook/webapp/components/QueryEditor/QueryEditor.tsx

```diff
@@ -130,7 +130,7 @@ export const QueryEditor: React.FC<
             language,
             query: value,
         });
-        const autoCompleter = useAutoComplete(
+        const { autoCompleter, autoCompleterRef } = useAutoComplete(
```

Comment:
seems `autoCompleter` is never used now, why it needs to be returned from `useAutoComplete`? and it's actually equals to autoCompleterRef.current